.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+jobs:
+  run-tests:
+    docker:
+      - image: circleci/python:latest
+    steps:
+      - checkout
+      - run:
+          name: Install Python requirements
+          command: |
+              pip install virtualenv
+              virtualenv ~/venv
+              source ~/venv/bin/activate
```

Comment:
Do we really need to setup a virtual environment for this? We're only using one application here anyway. Just curious if I'm missing something here, there's no harm at all in setting it up this way.

Reply:
I think this is a generally a good idea, since you can ensure you have exactly what you need and other libraries installed on the systems are not interfering somehow. I don't know that it's necessary here.